design-system/packages/core/package.json

```diff
@@ -16,6 +16,7 @@
   },
   "dependencies": {
     "@babel/runtime": "^7.13.10",
+    "@emotion/cache": "^10.0.27",
```

Comment:
Why did you add this?

Reply:
I was getting CI failures in the `document` field and the `docs-next` page. It seems like these was an implicit dependency being provided by one of the `@arch-ui` packages, which I traced back this far.

Not sure if this is the right solution, but so far it seems to at least make our tests pass 🤷‍♂️ .